Makefile

```diff
@@ -134,32 +138,28 @@ start-devstack: ## run a local development copy of the server
 	docker-compose --x-networking up
 
 open-devstack: ## open a shell on the server started by start-devstack
-	docker exec -it edx-exams /edx/app/edx-exams/devstack.sh open
+	docker exec -it edx_exams /edx/app/edx-exams/devstack.sh open
 
-pkg-devstack: ## build the edx-exams image from the latest configuration and code
-	docker build -t edx-exams:latest -f docker/build/edx-exams/Dockerfile git://github.com/edx/configuration
+pkg-devstack: ## build the edx_exams image from the latest configuration and code
+	docker build -t edx_exams:latest -f docker/build/edx-exams/Dockerfile git://github.com/edx/configuration
 
 detect_changed_source_translations: ## check if translation files are up-to-date
 	cd edx_exams && i18n_tool changed
 
 validate_translations: fake_translations detect_changed_source_translations ## install fake translations and check if translation files are up-to-date
 
 docker_build:
-	docker build . -f Dockerfile -t openedx/edx-exams
-	docker build . -f Dockerfile --target newrelic -t openedx/edx-exams:latest-newrelic
```

Comment:
There is no `newrelic` target in the Dockerfile. It was removed in this [PR](https://github.com/openedx/edx-cookiecutters/pull/173) from the cookiecutter. I'll plan to remove it from the cookiecutter soon.